doc-build/md054.md

```diff
@@ -0,0 +1,12 @@
+Links and images in Markdown can provide the link destination or image source
+at the time of use or can define it elsewhere and use a label for reference.
+The reference format is convenient for keeping paragraph text clutter-free
+and makes it easy to reuse the same URL in multiple places.
+
+This rule can be used to enforce a link style that is either:
+
+- provides the link destination or image source at the time of use (`inline`)
```

Comment:
Let's start with the style name since it's the most important:
- `inline`: Provides ...